src/main/java/uk/gov/hmcts/reform/sscs/transformers/SscsCaseTransformer.java

```diff
@@ -39,9 +35,37 @@ public CaseTransformationResponse transformExceptionRecordToCase(Map<String, Obj
     }
 
     private Appeal buildAppealFromData(Map<String, Object> pairs) {
+
+        Appellant appellant = null;
+
+        if (hasPerson(pairs, "person2")) {
+            Appointee appointee = null;
+            if (hasPerson(pairs, "person1")) {
+                appointee = Appointee.builder()
+                    .name(buildPersonName(pairs, "person1"))
+                    .address(buildPersonAddress(pairs, "person1"))
+                    .contact(buildPersonContact(pairs, "person1"))
+                    .identity(buildPersonIdentity(pairs, "person1"))
+                .build();
+            }
+            appellant = Appellant.builder()
+                .name(buildPersonName(pairs, "person2"))
+                .address(buildPersonAddress(pairs, "person2"))
+                .identity(buildPersonIdentity(pairs, "person2"))
+                .appointee(appointee)
+                .build();
+        } else if (hasPerson(pairs, "person1")) {
+            appellant = Appellant.builder()
+                .name(buildPersonName(pairs, "person1"))
```

Comment:
person1 and person2 could you please extract them to a constant ?

Reply:
done